packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -39,10 +39,15 @@ function loadDummyServerCertsIfNeeded() {
   dummyServerTlsOptions = { key, cert };
 }
 
+type ALPNCacheOptions = {
+  socket?: stream.Duplex | undefined;
```

Comment:
nit: we would drop `| undefined` and use `?` notation for both properties unless this would be a refactoring that needs to update all the call sites.